js/localgov_directories_entry_search.js

```diff
@@ -32,6 +32,15 @@
           label.childNodes[0].value = previousId;
         });
       });
+
+      // Add a bact to search results page if the referrer is the same path minus the last /page.
+      if (document.URL.substr(0, document.URL.lastIndexOf('/')) == document.referrer.split('?')[0]) {
+        var searchForm = document.getElementById('block-localgov-directories-channel-search-block');
```

Comment:
`querySelector` is a more modern approach here, and also not using `var` as that can be reassigned and cause issues. Maybe change to:

`const searchForm = document.querySelector('#block-localgov-directories-channel-search-block')`